external-import/sentinel-incidents/src/sentinel_incidents_connector/connector.py

```diff
@@ -61,11 +62,14 @@ def _get_last_incident_date(self) -> int:
         state = self.helper.get_state()
         if state and "last_incident_timestamp" in state:
             last_timestamp = state["last_incident_timestamp"]
-        elif self.config.import_start_date:
-            last_timestamp = self.config.import_start_date
-        else:
-            last_timestamp = 0
-        return last_timestamp
+            return last_timestamp
+
+        if self.config.import_start_date:
```

Comment:
Initially, the state should not be modified in this way, as this would risk causing incompatible changes for existing users already using the connector. I've therefore taken care to respect the current structure of the state.

As far as readability is concerned, for ‘External-Import’ connectors, the scheduler updates the ‘Last run’ and ‘Next run’ fields on the Front side. These fields are thus displayed in a ‘human-readable’ format to make them easier to understand.